### PR TITLE
Implement toggle for displaying thumbnail image in the background 

### PR DESCRIPTION
**Pull request by fheck**
I looked into implementing it myself and found it relatively easy. I hope I found everything I needed to adapt. If there is anything I missed, just tell me and I will try to fix it.

Implements #224

### CODE REVIEW

**Comment by ShankarBUS** [comment]
Wow!

Thank you so much mate! I was distracted away by some other things. Thanks again.

I will look into it as the first thing in the morning. Keep this for now ❤

File: ModernFlyouts/Controls/SessionControl.xaml.cs [hunk 1]
```diff
@@ -21,12 +21,25 @@ public partial class SessionControl : UserControl
                 typeof(bool),
                 typeof(SessionControl),
                 new PropertyMetadata(true, OnAlignThumbnailToRightChanged));
+        
+        public static readonly DependencyProperty ShowThumbnailInBackgroundProperty =
+            DependencyProperty.Register(
+                nameof(ShowThumbnailInBackground),
+                typeof(bool),
+                typeof(SessionControl),
+                new PropertyMetadata(true, OnShowThumbnailInBackgroundChanged));
```
Comment (ShankarBUS) [inline ModernFlyouts/Controls/SessionControl.xaml.cs:30]: These are not necessary. You could easily bind the **UIManager.ShowGSMTCThumbnailInBackground** property to the visibility of the border over here

https://github.com/ModernFlyouts-Community/ModernFlyouts/blob/a059480a40546b951b3396dce7da17cb624d4fea/ModernFlyouts/Controls/SessionControl.xaml#L25

Like this
```xaml
<UserControl ... xmlns:local="clr-namespace:ModernFlyouts" ...
....
<Border Grid.ColumnSpan="3" Grid.RowSpan="2"
    Opacity="0.5" CornerRadius="{DynamicResource FlyoutCornerRadius}"
    ClipToBounds="True"
    Visibility="{Binding Source={x:Static local:FlyoutHandler.Instance}, Path=UIManager.ShowGSMTCThumbnailInBackground, Converter={StaticResource BooleanToVisibilityConverter}}">
...
```

File: ModernFlyouts/Controls/SessionControl.xaml.cs [hunk 2]
```diff
@@ -21,12 +21,25 @@ public partial class SessionControl : UserControl
...
         {
             get => (bool)GetValue(AlignThumbnailToRightProperty);
             set => SetValue(AlignThumbnailToRightProperty, value);
         }
+        
+        public bool ShowThumbnailInBackground
+        {
+            get => (bool)GetValue(ShowThumbnailInBackgroundProperty);
+            set => SetValue(ShowThumbnailInBackgroundProperty, value);
+        }
```
Comment (ShankarBUS) [inline ModernFlyouts/Controls/SessionControl.xaml.cs:42]: Not needed see prev comment.

File: ModernFlyouts/Controls/SessionControl.xaml.cs [hunk 3]
```diff
@@ -41,6 +54,9 @@ public SessionControl()
             AlignThumbnailToRight = FlyoutHandler.Instance.UIManager.AlignGSMTCThumbnailToRight;
             BindingOperations.SetBinding(this, AlignThumbnailToRightProperty,
                 new Binding(nameof(UI.UIManager.AlignGSMTCThumbnailToRight)) { Source = FlyoutHandler.Instance.UIManager });
+            ShowThumbnailInBackground = FlyoutHandler.Instance.UIManager.ShowGSMTCThumbnailInBackground;
+            BindingOperations.SetBinding(this, ShowThumbnailInBackgroundProperty,
+                new Binding(nameof(UI.UIManager.ShowGSMTCThumbnailInBackground)) { Source = FlyoutHandler.Instance.UIManager });
```
Comment (ShankarBUS) [inline ModernFlyouts/Controls/SessionControl.xaml.cs:59]: Not needed see prev comment.

File: ModernFlyouts/Controls/SessionControl.xaml.cs [hunk 4]
```diff
@@ -175,6 +191,20 @@ private void EndTrackTransition()
...
+            if (showThumbnailInBackground)
+            {
+                sessionControl.ThumbnailBackgroundBrush.ImageSource = sessionControl._mediaSession.Thumbnail;
+            }
+            else
+            {
+                sessionControl.ThumbnailBackgroundBrush.ImageSource = null;
+            }
+        }
+
```
Comment (ShankarBUS) [inline ModernFlyouts/Controls/SessionControl.xaml.cs:207]: Not needed see prev comment

**Review by ShankarBUS — CHANGES_REQUESTED** [review]
Could you please correct those suggestions?
Or else I could do that after merging.

**Comment by ShankarBUS** [comment]
Wow! That was quick.

@Samuel12321, merge this marvel!

**Comment by fheck** [comment]
I have made the requested changes. I have some experience with C#, but I haven't done much with xmal, so I wasn't aware there is a simpler way.
That was a fun evening project, so I just did it now😃.

**Comment by ShankarBUS** [comment]
Also thanks for the free translations lol. Saves us some time.

**Comment by fheck** [comment]
I only speak English and German, so I can't help with the other ones, but I'm happy to do those I know.

**Review by Samuel12321 — APPROVED** [review]
Looks good.

Thanks so much for your help

**Pull request merged**